packages/node_modules/@webex/plugin-meetings/src/locus-info/mediaSharesUtils.js

```diff
@@ -0,0 +1,89 @@
+import {CONTENT} from '../constants';
+
+const mediaSharesUtils = {};
+
+mediaSharesUtils.parse = (mediaShares) => {
+  if (mediaShares) {
+    return {
+      contentId: mediaSharesUtils.getContentId(mediaShares),
+      disposition: mediaSharesUtils.getDisposition(mediaShares)
+    };
+  }
+  return null;
```

Comment:
Would an empty object make more sense here? 

I pose this not as a suggestion, but more as a prompt. Are we using `null`s consistently?

Reply:
we are using nulls consistently